assets/js/group-details.js

```diff
@@ -0,0 +1,467 @@
+/* global jQuery:false, wpApiSettings:false */
+
+jQuery.ajaxSetup({
+  beforeSend: function(xhr) {
+    xhr.setRequestHeader('X-WP-Nonce', wpApiSettings.nonce);
+  },
+})
```

Comment:
Note this has implications for all Javascript globally, even Javascript outside this file. This may or may not be a good thing.

Reply:
@burntsun this file is only loaded on the group details page. I have the same code in contacts details and it did not apply to the group details.

Reply:
@burntsun  this file is only loaded on the group details page. I have the same code in contacts details and it did not apply to the group details.

Reply:
Yes, this file is only loaded on one page, but the code `xhr.setRequestHeader` applies to all jQuery AJAX calls made on that page, even in other scripts. It has global effects. I would prefer it if it only affected code within the file `group-details.js`.